src/sonic-config-engine/tests/test_cfggen.py

```diff
@@ -13,8 +13,12 @@ def setUp(self):
         self.sample_graph_metadata = os.path.join(self.test_dir, 'simple-sample-graph-metadata.xml')
         self.sample_graph_pc_test = os.path.join(self.test_dir, 'pc-test-graph.xml')
         self.sample_graph_bgp_speaker = os.path.join(self.test_dir, 't0-sample-bgp-speaker.xml')
+        self.sample_graph_t2_chassis_fe = os.path.join(self.test_dir, 't2-chassis-fe-graph.xml')
```

Comment:
i'd suppose it is better to separate the t2 chassis tests into one separate test file since it introduces many new features

Reply:
@stcheng  Fixed